web/static/js/command-palette.js

```
@@ -0,0 +1,339 @@
+    row.className = 'sw-cmdk-row';
+    row.setAttribute('role', 'option');
+    row.setAttribute('data-idx', String(idx));
+    row.setAttribute('data-id', item.id);
+
+    var label = document.createElement('span');
+    label.className = 'sw-cmdk-row-label';
+    label.textContent = (item.id && item.id === armedConfirmId) ? CONFIRM_LABEL : item.label;
+    row.appendChild(label);
+
+    if (item.kind === 'screen' && item.shortcut && item.shortcut.length === 2) {
+      var chips = document.createElement('span');
```

Biome (2.5.1)

[error] This var should be declared at the root of the enclosing function.

(lint/correctness/noInnerDeclarations)